hasura/migrations/default/1705344968980_modify-app-metadata-functions/up.sql

```diff
@@ -29,9 +29,9 @@ CREATE FUNCTION set_is_row_verified()
 RETURNS TRIGGER AS $$
 BEGIN
   IF NEW.verification_status = 'verified' THEN
-    NEW.is_row_verified := false;
-  ELSE
     NEW.is_row_verified := true;
+  ELSE
+    NEW.is_row_verified := false;
```

Comment:
let's edit the function in the new migration, this migration is likely already applied